spec/requests/volume_mappings_spec.rb

```diff
@@ -54,6 +54,51 @@
     end
   end
 
+  it "can delete a single volume mapiing" do
+    volume_mapping = FactoryBot.create(:volume_mapping)
+
+    api_basic_authorize("volume_mapping_delete")
```

Comment:
Shouldn't this be using `collection_action_identifier` rather than the hard-coded feature?